daemon/network.go

```diff
@@ -315,9 +315,22 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		driver = c.Config().DefaultDriver
 	}
 
+	networkOptions := make(map[string]string)
+	for k, v := range create.Options {
+		networkOptions[k] = v
+	}
+	if defaultOpts, ok := daemon.configStore.DefaultNetworkOpts[driver]; ok {
+		for k, v := range defaultOpts {
+			logrus.Debugf("Applying default option %v=%v to network %v", k, v, id)
```

Comment:
nit: should we consider including the driver, and using structured logs for this?

```go
logrus.WithFields(logrus.Fields{"driver": driver, "network": id, k: v}).Debug("Applying default option")
```

Reply:
:heavy_check_mark: